frontend/cypress/e2e/Pages/User/UserCreation.feature

```diff
@@ -0,0 +1,18 @@
+Feature: User Creation
+
+Scenario: IDIR user creates a new user
```

Comment:
praise: Such a clean way or writing the requirements!

Reply:
Thanks, Daniel, for reviewing my PR!

It's a great way to make the testing process easier to understand for both technical and non-technical people.

I removed the waits and It works well! It will speed up the tests, making everything more efficient. Appreciate the feedback! 👍